lib/child_process.js

```diff
@@ -297,12 +297,14 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
-var _deprecatedCustomFds = internalUtil.deprecate(function(options) {
-  options.stdio = options.customFds.map(function(fd) {
-    return fd === -1 ? 'pipe' : fd;
-  });
-}, 'child_process: options.customFds option is deprecated. ' +
-   'Use options.stdio instead.');
+const _deprecatedCustomFds = internalUtil.deprecate(
+  function deprecateCustomFds(options) {
+    options.stdio = options.customFds.map(function(fd) {
```

Comment:
@lpinca The goal would be getting rid of the anonymous callback functions, so if we were to go the route of an arrow function it'd be declaring a new const above this declaration that defines the callback function, then passing that const to internalUtil.deprecate. I'm :+1: on doing that, but I want to make sure that that would fit the overall code style/design patterns we're wanting to achieve.